sites/docs/docs/about.md

```diff
@@ -6,18 +6,6 @@ slug: /about
 
 Vitessce is developed by the [Gehlenborg Lab](http://gehlenborglab.org/research/projects/vitessce/) at Harvard Medical School.
 
-## Showcase
-
-Vitessce is being used in the following projects:
-
-* [HuBMAP Data Portal](https://portal.hubmapconsortium.org/)
-* [BaysorAnalysis](https://github.com/kharchenkolab/BaysorAnalysis/tree/68f568d1c056708d474aeea3f28bbe083ccb6e4c#visualization-of-the-results)
-* [Azimuth](https://azimuth.hubmapconsortium.org/references/human_pbmc/)
-* [KPMP Kidney Tissue Atlas Spatial Viewer](https://atlas.kpmp.org/spatial-viewer/)
-* [cBioPortal](https://www.thehyve.nl/articles/single-cell-data-visualisation-in-cbioportal)
-* [scimap_phenotyping](https://github.com/ohsu-comp-bio/scimap_phenotyping/tree/2d0e63)
-* [Cellenics](https://github.com/hms-dbmi-cellenics/ui/blob/394a115/src/components/data-exploration/heatmap/HeatmapPlot.jsx#L30)
```

Comment:
This is unrelated?  Just checking

Reply:
I will remove some unrelated things - I went overboard here